src/mouse_events.hpp

```diff
@@ -87,6 +87,18 @@ class mouse_handler : public mouse_handler_base {
 
 	void select_or_action(bool browse);
 
+	/// Wraps 'const map_location game_display::hex_clicked_on(int, int) const'.
```

Comment:
We prefer `/** Text */` formatting to `///`. Also applied to the other methods below.

Reply:
Will comply request.